src/components/interaction-designer/block-editors/EndRecordingDigitsEditor.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+  <div class="end-recording-digits-editor">
+    <validation-message
```

Comment:
Curiosity, where you able to test a scenario which shows a validation message? eg: empty value

Reply:
We don't have restrictions on `ivr.end_recording_digits` apart from _string_ type. I tried forcing a value using devtools, and the message looks as follows:

<img width="1080" alt="image" src="https://user-images.githubusercontent.com/3971957/170680495-0921712c-ebe2-4d18-8fff-1f6a98522564.png">